passes/cmds/add.cc

```diff
@@ -53,7 +52,6 @@ static void add_wire(RTLIL::Design *design, RTLIL::Module *module, std::string n
 		wire->port_output = flag_output;
 
 		if (flag_input || flag_output) {
-			wire->port_id = module->wires_.size();
 			module->fixup_ports();
 		}
```

Comment:
As far as I can tell, the assignment to `wire->port_id` is superfluous, as the immediately following call to `module->fixup_ports()` will re-assign all `port_id`s for all module wires that have `port_input` or `port_output`.
So rather than update this section verbatim simply to avoid using `module->wires_` I decided just to remove that assignment.